code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -118,11 +116,9 @@
 		evolve()
 		return
 
-/// This proc is used to determine if a mouse should chew on a wire or not. Returns TRUE if it should, FALSE if it shouldn't.
-/mob/living/simple_animal/mouse/proc/should_chew()
-	if(prob(15) || always_chew)
-		return TRUE
-	return FALSE
+/// Proc to pass the probability of a mouse chewing on a wire to handle_automated_action. Overriden on applicable subtypes.
+/mob/living/simple_animal/mouse/proc/chew_probability()
```

Comment:
no no keep this as `should_chew` that name is much better

Reply:
I don't think `should_chew()` is a good name because the whole automated action chain has like three things a mouse could potentially chew on (cheese, royal cheese, wires) and it seems to imply that it toys with the `var/chew_probability` that mice have (called at the very beginning of the automated action chain). I think incorporating the fact that it directly pertains to cables (since we only call it in relation to cables) is a good idea.

Reply:
But it's not the probability, it's a bool. `chew_probability()` implies it returns the percent chance to me.

Name the action whatever but it needs to be `should_action()` if it returns a bool.